db/migrate/20230117190802_create_workflow_migrations.rb

```diff
@@ -0,0 +1,35 @@
+class CreateWorkflowMigrations < ActiveRecord::Migration[6.1]
+  def change
+    create_table :workflows do |t|
+      t.references :ems,           :type => :bigint, :index => true, :references => :ext_management_system
+      t.references :tenant,        :type => :bigint, :index => true
+      t.references :git_reference, :type => :bigint, :index => true
+      t.references :evm_owner,     :type => :bigint, :index => true, :references => :user
+      t.references :miq_group,     :type => :bigint, :index => true
```

Comment:
Do we need tenant here?

Reply:
I thought we needed both per this comment https://github.com/ManageIQ/manageiq-schema/pull/679/files#r1132272597

Reply:
Oh sorry - didn't see it 3 lines up